tools/snapshot_tool/src/main.rs

```diff
@@ -58,9 +58,7 @@ fn main() {
         .get_matches();
     let command = matches.value_of("cmd").expect("provide specific command");
     let command = cast_command(command);
-    let file = matches
-        .value_of("file")
-        .expect("provice specific snapshot-file path");
+    let file = matches.value_of("file").unwrap_or("snapshot");
```

Comment:
User get error message:
```
output: -bash: ../../bin/snapshot_tool: cannot execute binary file
```
Why this change can fix the bug? Has user run `panic` ?

Reply:
Noop! 
This change solves `ls snap*` no result problem. Using default file path config, snapshot tool will panic.

`cannot execute binary file` it occured only in mac，caused by different platforms.

Reply:
So it's nothing about the #496

> cannot execute binary file it occured only in mac，caused by different platforms.

Update the conclusion into the #496.